server/scripts/2021TaxSeasonDataScript.php

```diff
@@ -0,0 +1,420 @@
+<?php
+// This script will only work for Joey Carrigan.
+// To run it locally, make sure userId and siteIds exist here and in helper functions
+
+$root = realpath($_SERVER['DOCUMENT_ROOT']);
+require_once "$root/server/user.class.php";
+$USER = new User();
+if (!$USER->isLoggedIn() || $USER->getUserId() !== '358') {
+	header("Location: /unauthorized");
+	die();
+}
+
+// TODO: Need to update the `$dataAlreadyInserted` values before this query will run successfully
+
+insert2021Data();
+
+function insert2021Data() {	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The data has already been inserted');
+	}
+//	insertAndersonLibraryData();
+//	insertCenterForPeopleInNeedData();
+//	insertLorenEiseleyLibraryData();
+//	insertBennettMartinLibraryData();
+//	insertFStreetCommunityCenterData();
+//	insertSoutheastCommunityCollegeData();
+//	insertAsianCommunityAndCulturalCenterData();
+//	insertNebraskaUnionData();
+//	insertInternationalStudentScholarSiteData();
+	
+	die('SUCCESS');
+}
+
+function insertAndersonLibraryData() {
+	GLOBAL $DB_CONN;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Anderson Library data has already been inserted');
+	}
+
+	$siteId = 2; // Manually obtained from PROD DB
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		// Wednesdays 5-8
+		$wednesdayDates = getWeeklyDatesFromRange('2021-02-03', '2021-04-07');
+		foreach ($wednesdayDates as $date) {
+
+			// Are from Katie's parameter sheet
+			// 6 is Virtual Residential
+			$firstAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 7.5, 6, $siteId);
```

Comment:
What's with "7.5" appointments--is that valid?

Reply:
It's 1.5 taxes/hour for 5 preparers--it will convert to 8 in the DB but I could make it explicitly 8.

Reply:
I made it 8